org.eclipse.scanning.api/src/org/eclipse/scanning/api/points/MapPosition.java

```diff
@@ -68,8 +68,8 @@ public MapPosition(String name, Integer index, Object value) {
 	 */
 	public MapPosition(String value) throws NumberFormatException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.NumberFormatException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)